src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -357,6 +357,51 @@ await foreach (var streamValue in asyncEnumerable)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(HubProtocolsAndTransportsAndHubPaths))]
+        [LogLevel(LogLevel.Trace)]
+        public async Task IAsyncEnumerableServerParam(string protocolName, HttpTransportType transportType, string path)
+        {
+            var protocol = HubProtocols[protocolName];
+            using (StartServer<Startup>(out var server))
+            {
+                var connection = CreateHubConnection(server.Url, path, transportType, protocol, LoggerFactory);
+                try
+                {
+                    async IAsyncEnumerable<int> clientStreamData()
```

Comment:
Casing!

Reply:
Too much Java. This doesn't look wrong anymore 😅

Reply:
We need enforced editorconfig to save us!

Reply:
Didn't VS2019 get something like this that we can check in to github?